context/app/static/js/components/workspaces/LaunchWorkspaceDialog/hooks.ts

```diff
@@ -111,6 +113,22 @@ function useLaunchWorkspaceDialog() {
     ],
   );
 
+  const handleLaunch = useCallback(
+    (currentWorkspace: MergedWorkspace) => {
+      setWorkspace(currentWorkspace);
+
+      if (currentWorkspace.jobs.length > 0) {
+        submit({ workspaceJobTypeId: currentWorkspace.jobs[0].job_type }).catch((e) => {
```

Comment:
Can we create a new util which perhaps uses `findBestJob` which returns the workspace's `job_type`?